src/screens/docs/components/victory-errorbar/ecology.md

```diff
@@ -213,7 +213,7 @@ The `events` prop takes an array of event objects. Event objects are composed of
 
 ### eventKey
 
-The `eventKey` prop is used to assign eventKeys to data. This prop operates identically to the `x` and `y` data accessor props. By default, the eventKey of each datum will be equal to its index in the data array. `eventKey` may also be defined directly on each data object.
+The `eventKey` prop is used to assign 'eventKeys' to data. This prop operates identically to the `x` and `y` data accessor props. By default, the 'eventKey' of each datum will be equal to its index in the data array. `eventKey` may also be defined directly on each data object.
```

Comment:
hey @mindjuice! thanks for your help on this. I'd love to merge this in. Do you have time to make those back ticks?